src/Kernels/Receiver.h

```diff
@@ -88,7 +88,7 @@ namespace seissol {
           seissolInstance(seissolInstance)
       {}
 
-      ReceiverCluster(  GlobalData const*             global,
+ReceiverCluster(  GlobalData const*             global,
```

Comment:
Why did you change this? Please keep the diff of the PR as small as possible. Otherwise the git history of this file will be a nightmare to read. In a year, nobody will understand, why this line was changed.

Reply:
It must've been some formatting error. I applied the clang-format again on the header and .cpp file. I hope the diff is not too different now